util/minetestmapper.py

```diff
@@ -190,22 +194,126 @@ def usage():
         draworigin = True
     elif o == "--drawunderground":
         drawunderground = True
+    elif o == "--geometry":
+        geometry_string = a
+    elif o == "--region":
+        region_string = a
     else:
         assert False, "unhandled option"
 
+nonchunky_xmin = -1500
+nonchunky_xmax = 1500
+nonchunky_zmin = -1500
+nonchunky_zmax = 1500
+
+if geometry_string is not None:
+    parts = geometry_string.split("+")
+    if len(parts) == 3:
+        coords_string, width_string, height_string = parts
+        coords = coords_string.split(":")
+        if len(coords) == 2:
+            x_string, z_string = coords
+            x = int(x_string)
+            z = int(z_string)
+            this_width = int(width_string)
+            this_height = int(height_string)
+            nonchunky_xmin = x
+            nonchunky_xmax = nonchunky_xmin + this_width - 1  # inclusive rect
+            nonchunky_zmin = z
+            nonchunky_zmax = nonchunky_zmin + this_height - 1  # inclusive rect
+            print("#geometry:")
```

Comment:
If you are saying something about the '#'s being printed, that is so the output is machine-readable (always prints region notation, and prints comments with values from geometry notation if that was the source). I can't call print once using a multiline comment (string) since there are variables. I'm not sure what you mean. Do you mean formatted multiline string like:
```python
print("""
#geometry:
#    x:{}
#    z:{}
#    width:{}
#    height:{}
region:
  xmin:{}
  xmax:{}
  zmin:{}
  zmax:{}
                  """.format(x,z,this_width,this_height,
                             nonchunky_xmin,nonchunky_xmax,
                             nonchunky_zmin,nonchunky_zmax))
```
(leaving indents off of the string as done above would be necessary due to the fact they would be printed otherwise--the way around this would be adding the lines together, keeping the string conversions [or format call for each line] and putting a newline at the end of each instead of using a multiline string)